.github/workflows/_data-plane.yml

```diff
@@ -339,8 +344,24 @@ jobs:
           gh release upload ${{ matrix.name.release_name }} \
             "$BINARY_DEST_PATH" \
             "$BINARY_DEST_PATH".sha256sum.txt \
+            # "$BINARY_DEST_PATH".deb \ # Enable this once we have all the necessary documentation in place.
```

Comment:
@jamilbk I've commented this out for now which means users should not yet see that we are even building a `.deb` file. That should make this PR safe to merge which allows us to test things using the preview repository.

We can then create all the supporting documentation later.

Reply:
👍